configs/machines/generic.yaml

```diff
@@ -3,6 +3,8 @@
 name: generic
 accounting: false
 
+
+hetjob_flag: "?" # either hetjob or packjob
```

Comment:
We should consider turning the `generic.yaml` file into something we can include in the documentation page with comments that show what you would need to do to add a new machine.

As things such as `hetjob` may depend on the scheduler version, maybe we can also include a variable for `batch_system_version`. It will probably never be actually used, but it might be good as a place to look as a reminder.

Reply:
I think this is better handled manually. Adding another choose for something that only changes effectively between computers seems a bit of an overkill to me.